docker/Dockerfile.workers_web

```diff
@@ -0,0 +1,78 @@
+# --- build essentia_streaming_extractor_freesound static binary
+
+FROM debian:10 as essentia
+# This is basically the contents of https://github.com/MTG/essentia-docker
+# but it only builds the freesound extractor as a static binary
+
+RUN apt-get update \
+    && apt-get install -y \
+       python libpython2.7 build-essential git wget pkg-config nasm cmake curl \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /essentia && cd /essentia && git clone https://github.com/MTG/essentia.git
+WORKDIR /essentia/essentia
+# Checkout commit with Freesound Extractor update 0.6
+RUN git fetch && git checkout 89f52d4efb21e11d3745fa4ce4e0bb0e49cd0e98
+
+WORKDIR /essentia/essentia/packaging
+RUN ./build_3rdparty_static_debian.sh
+
+WORKDIR /essentia/essentia/
+# The line below is a dirty hack to fix a bug currently in Essentia's master branch (and merged to 89f52d4efb21e11d3745fa4ce4e0bb0e49cd0e98)
+# This can be removed once we update the commit to checkout for a version without the bug
+RUN sed -i 's/"HAVE_TENSORFLOW"/0/' src/examples/wscript
+
+RUN PKG_CONFIG_PATH=packaging/debian_3rdparty/lib/pkgconfig ./waf configure --with-static-examples --with-example=streaming_extractor_freesound \
+    && ./waf && ./waf install
+
+# --- build stereofy static binary
+
+FROM debian:10 as stereofy
+
+RUN apt-get update && apt-get install -y build-essential libsndfile-dev
+
+RUN mkdir /code
+COPY ./_sandbox/stereofy /code
+WORKDIR /code/
+RUN make clean && make
+
+# --- main Freesound docker file contents
+
+FROM freesound:2020-02
+
+# Install spcific dependencies needed for processing
+RUN apt-get update \ 
+	&& apt-get install -y --no-install-recommends \ 
+		sndfile-programs \
+		libsndfile1-dev \
+		mplayer \
+		lame \
+		vorbis-tools \
+		flac \
+		faad \
+		libjpeg-dev \
+		zlib1g-dev \
+		libpng-dev \
+	&& rm -rf /var/lib/apt/lists/*	
+
+RUN mkdir /code
+RUN mkdir /freesound-data
+WORKDIR /code
+
+# Copy essentia_streaming_extractor_freesound and stereofy binaries
+COPY --from=essentia /usr/local/bin/essentia_streaming_extractor_freesound /usr/local/bin
+COPY --from=stereofy /code/stereofy /usr/local/bin
+
+# Install python dependencies
+COPY --chown=fsweb:fsweb requirements.txt /code/requirements.txt
+RUN pip install --no-cache-dir -r /code/requirements.txt
+
+# To avoid installation problems, scikits.audiolab needs to be installed after the other packages in requirements.txt
+# have been installed. We might want to replace this package in the future to avoid this "hack".
+RUN pip install --no-cache-dir scikits.audiolab==0.11.0
+
+# Copy source code
+COPY --chown=fsweb:fsweb . /code
+
+# NOTE: this image should be build from the root directory of the repository so the correct source files are coppied
```

Comment:
if we have tools to build this image I don't think we need this comment

Reply:
ok. I'll remove it from other files as well